lang/cs/Org.Apache.REEF.IO.Tests/TestAzureDataLakeFileSystem.cs

```diff
@@ -13,8 +13,8 @@ namespace Org.Apache.REEF.IO.Tests
     public sealed class TestAzureDataLakeFileSystem
     {
         private readonly static Uri FakeBaseUri = new Uri("http://fakeadls.com");
-        private readonly static Uri FakeFileUri = new Uri("http://fakeadls.com/dir/fakefile");
-        private readonly static Uri FakeDirUri = new Uri("http://fakeadls.com/dir/");
+        private readonly static Uri FakeDirUri = new Uri(FakeBaseUri, "dir");
+        private readonly static Uri FakeFileUri = new Uri($"{FakeDirUri}/fakefile");
```

Comment:
Why is this not following the same pattern as FakeDirUri?

Reply:
uri constructor takes the first argument as only the baseUri and appends the rest to the base Uri
So FakeFileUri = new Uri(FakeDirUri, "fakefile"); will give http://fakeadls.com/fakefile and remove dir